src/libraries/Externals.csproj

```diff
@@ -1,44 +1,48 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.Build.NoTargets">
   <PropertyGroup>
+    <!-- Set the RuntimeIdentifier so that the DotNetHost and DotNetHostPolicy packages resolve for the corresponding runtime. -->
     <RuntimeIdentifier>$(PackageRID)</RuntimeIdentifier>
-    <NoWarn>$(NoWarn);NU1603;NU1605</NoWarn>
     <SwapNativeForIL Condition="'$(SwapNativeForIL)' == '' and ('$(Configuration)' == 'Debug' or '$(Coverage)' == 'true') and '$(RuntimeFlavor)' != 'Mono'">true</SwapNativeForIL>
+    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
```

Comment:
Will we run into clashing when folks run different vertical builds?  Like building for browser and unix from same clone?

Reply:
I just verified this scenario, the outputs aren't clashing:

![image](https://user-images.githubusercontent.com/7412651/92410007-f348ee00-f142-11ea-883f-30c1b02685ec.png)

The Browser leaf contains the mono aot stuff and the Linux leaf contians the host and other stuff that isn't present on wasm.